### DISCUSSION TITLE
Authenticated login doesn't work on kali linux

### DISCUSSION THREAD

**Discussion by veganjoe** [gavvvr/obsidian-imgur-plugin#71] [Getting help]
It pops open firefox, and I can authenticate and click allow, but it never successfully passes the auth back to the obsidian imgur plugin.

**Comment by greyyT** [comment]
I also get this error, hope you can guide me on how to fix it. Thanks.

**Comment by Sohail-XD** [comment]
Same issue. Seems like proxy is the problem. Disabling google's 8.8.8.8 dns over https proxy fixed the issue.

**Comment by gavvvr** [comment]
I've just tried to use the plugin with Debian + **xfce** DE. If you run Obsidian from the terminal, it mentions that:
> xdg-settings: default-url-scheme-handler not implemented for xfce

I guess it's the root cause of the problem. After successful authentication browser tries to redirect you to `obsidian://imgur-oauth`, but Obsidian running within xfce does not handle it. The problem is definitely not related to the plugin. Not sure how can I help you here. It's either the problem of xfce or Obsidian
